### ISSUE TITLE
Checkstyle config enforces line break at "do { ... }|while(...);"

### ISSUE THREAD

**Issue by dflemstr** [spotify/spotify-checkstyle-config#12]
The config rejects this code:

```java
do {
  System.out.println(1);
} while (true);
```

It accepts the following code:

```java
do {
  System.out.println(1);
}
while (true);
```

However, the first example matches the [Google style guide](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style) while the second one does not:

> Line break after the closing brace, only if that brace terminates a statement or terminates the body of a method, constructor, or named class. For example, there is no line break after the brace if it is followed by else or a comma.

Opening this as an Issue as I'm not familiar with the Checkstyle config syntax so I don't know the idiomatic way to fix it, and I want to give people a chance to bikeshed about this ;)

**Comment by davidxia** [comment]
The first is better and was fixed by Google here https://github.com/checkstyle/checkstyle/pull/3679/files#diff-c55cd603ef6597463971db3f33f4f4c8.

**Comment by caipre** [comment]
Fixed by #17.
